src/Redux/api.tsx

```diff
@@ -47,16 +54,22 @@ const routes = {
   checkResetToken: {
     path: "/api/v1/password_reset/check/",
     method: "POST",
+    TRes: Type<any>(),
+    TBody: Type<{ token: string }>(),
   },
 
   resetPassword: {
     path: "/api/v1/password_reset/confirm/",
     method: "POST",
+    TRes: Type<any>(),
+    TBody: Type<{ password: string; confirm: string }>(),
   },
 
   forgotPassword: {
     path: "/api/v1/password_reset/",
     method: "POST",
+    TRes: Type<any>(),
```

Comment:
For no response bodies, you can use this instead of any

```suggestion
    TRes: Type<Record<string, never>>(),
```